dictionary.txt

```diff
@@ -0,0 +1,2 @@
+rouge
+sart
```

Comment:
What is `sart`?

Reply:
a word tha ppears in the ipynb and that the codespell chokes on